src/init.cpp

```diff
@@ -1662,11 +1662,12 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
         static_cast<uint16_t>(args.GetIntArg("-port", Params().GetDefaultPort()));
 
     const auto BadPortWarning = [](const char* prefix, uint16_t port) {
-        return strprintf(_("%s request to listen on port %u. This port is considered \"bad\" and "
-                           "thus it is unlikely that any Bitcoin Core peers connect to it. See "
+        return strprintf(_("%s requests to listen on port %u. This port is considered \"bad\" and "
+                           "thus it is unlikely that any %s peers connect to it. See "
```

Comment:
```suggestion
                           "thus it is unlikely that any peers will connect to it. See "
```

No reason to name the software specifically here?

Reply:
true

Reply:
On second thought, we are touching this, but it isn’t a part of the pr’s goal to address something like this; seems appropriate for a follow-up if someone wants to suggest this change.

Reply:
> No reason to name the software specifically here?

[Updated](https://github.com/bitcoin/bitcoin/pull/26142#issuecomment-1260104579).